tests/app/services/test_response_formatters.py

```diff
@@ -17,18 +19,24 @@
     SEARCH_RESULTS_JSON = json.load(search_results)
 
 
-def test_should_build_query_block_in_response(services_mapping):
+@mock.patch('app.main.services.response_formatters.current_app')
+def test_should_build_query_block_in_response(current_app, services_mapping):
```

Comment:
I'm curious as to why this patch is needed in this commit? Was it missed before, or has something changed to make it needed when it wasn't before?

Reply:
Oh wait, I've realised now, ignore me 🤦